packages/tool/site/server/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "EPS API Tool Server",
   "main": "app.js",
   "scripts": {
-    "test": "jest --ci --reporters=default --reporters=jest-junit",
+    "test": "jest --ci --reporters=default --reporters=jest-junit --coverage  --coverageReporters=cobertura --coverageReporters=text --coverageReporters=html",
```

Comment:
I don't suppose these are still needed, if we're only publishing the client coverage?

Reply:
good point - removed